CLI/Operation/Processor.php

```diff
@@ -44,8 +44,8 @@ public function processOperations(Image $input_image, Image $output_image = null
         // Check for a SequenceNumber or Density operation.  They're a little special.
         foreach ($this->operations as $index => $operation) {
             if ($operation instanceof SequenceNumber) {
-                $frame = (int)$operation->getValue();
-                if (!$frame) {
+                $frame = $operation->getValue();
+                if (!is_int($frame)) {
```

Comment:
You've changed how the code functions.

```php
<?php

$frame_sz = "1";
$frame_i = 1;

var_dump(is_int($frame_sz), is_int($frame_i));

OUTPUT:

bool(false)
bool(true)
```

"1" was equally valid here.  is_int() checks the underlying zval, but casting to int retains the value.

If you're worried that a frame value like "FOO" will mess things up because it would cast to int 0, please use `is_numeric()` instead.